core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -142,7 +141,9 @@ class DagrCoreMain(
   @arg(doc = "Set the memory available to dagr.", common = true)
   val memory: Option[String] = None,
   @arg(doc = "Write an execution report to this file, otherwise write to the stdout", common = true)
-  val report: Option[Path] = None
+  val report: Option[Path] = None,
+  @arg(doc = "Provide an top-like interface for tasks with the give delay in seconds. This suppress info logging.")
+  val taskTop: Double = -1
```

Comment:
I think we can have the refresh rate configurable in the interface itself like top, so I can just make it a flag for now.